doc/developers/contributing.rst

```diff
@@ -872,6 +872,51 @@ from high-level questions to a more detailed check-list.
   instructive?
 
 
+Standard replies
+................
+
+It may be helpful to store some of these in GitHub's [saved
+replies](https://github.com/settings/replies/) for reviewing:
+
+Issue: Update the docs svp
```

Comment:
svp seems a little magical at least for me

Reply:
it's french, I assume ;)

Reply:
Okay. I won't use any french.